portfolio/src/main/webapp/script.js

```diff
@@ -39,33 +45,38 @@ function showSlides(newIndex) {
     slideIndex = newIndex;
   }
   // Display the photo.
-  slides[slideIndex].style.display = "block";
-  dots[slideIndex].className += " active";
+  slides[slideIndex].style.display = 'block';
+  dots[slideIndex].className += ' active';
 }
 
 /** Displays list of comments returned by the server.*/
 function showComments() {
-  const limit = document.getElementById("select-comment-number").value;
-  fetch('/comments?number=' + limit).then(response => response.json()).then(comments => {
-    const commentsList = document.getElementById('text-container');
-    commentsList.innerHTML = '';
-    for(let i = 0; i < comments.length; i++) {
-      commentsList.appendChild(createCommentElement(comments[i]));
-    }
-  });
+  const limit = document.getElementById('select-comment-number').value;
+  fetch('/comments?number=' + limit)
+      .then((response) => response.json()).then((comments) => {
+        const commentsList = document.getElementById('text-container');
+        commentsList.innerHTML = '';
+        for (let i = 0; i < comments.length; i++) {
+          commentsList.appendChild(createCommentElement(comments[i]));
+        }
+      });
 }
 
-/** Creates an <p> comment element containing text. */
+/**
+ * Creates an <p> comment element containing text.
+ * @param {string} text - The comment content.
+ * @return {HTMLElement} - The html element.
```

Comment:
Usually you write it without dash https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments